### DISCUSSION TITLE
How to update email id or mobile number of existing users

### DISCUSSION THREAD

**Discussion by ankitv89** [Sunbird-Lern/Community#108] [Query] [answered]
Hi,

We have a use case to allow the updation of email id/phone numbers of certain users by admin. I am not able to locate appropriate endpoints for that. Any suggestions on how to achieve this?

**Comment by BharathwajShankar** [comment, marked as answer]
@ankitv89 Please refer to the #66 thread for the above question. You can use /v1/user/upload API for the above operation
Please refer to the below documentation for the API.
http://docs.sunbird.org/latest/apis/bulkupload/
